hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/CompactionConfiguration.java

```diff
@@ -172,7 +173,9 @@ public String toString() {
       dateTieredIncomingWindowMin,
       compactionPolicyForDateTieredWindow,
       dateTieredSingleOutputForMinorCompaction,
-      dateTieredCompactionWindowFactory
+      dateTieredCompactionWindowFactory,
+      storeConfigInfo.getRegionInfo().getEncodedName(),
```

Comment:
You want to do RegionInfo.prettyPrint(storeConfigInfo.getRegionInfo().getEncodedName())?

Reply:
OK